demisto_sdk/dev_tools/spell_checker.py

```diff
@@ -1,95 +1,129 @@
 import yaml
-
+from typing import Set
 from spellchecker import SpellChecker
+
+from demisto_sdk.common.constants import KNOWN_WORDS
 from demisto_sdk.common.tools import print_error, print_color, LOG_COLORS
-from demisto_sdk.common.constants import DISPLAYABLE_LINES, SCRIPT_ARGS, KNOWN_WORDS
+
+
+# These are keys in a Demisto yml file which indicate that their values are visible to the user and
+# thus their spelling should be checked.
+USER_VISIBLE_LINE_KEYS = [
+    "description",
+    "display",
+    "name",
+    "comment"
+]
 
 
 class SpellCheck:
-    """
+    """Preform a spell check on the given .yml or .md file.
         Attributes:
-            path (str): The path to the current file being checked
+            path (str): The path to the current file being checked.
+            known_words (str): The path to a file containing known words.
+            spellchecker (SpellChecker): The spell-checking object.
+            unknown_words (set): A set of unknown words found in the given file.
     """
 
-    def __init__(self, path: str):
+    def __init__(self, path: str, known_words: str):
         self.path = path
```

Comment:
Let's change this name to checked_file_path

Reply:
done